libs/components/src/progress-bar/vl-progress-bar.component.cy.ts

```diff
@@ -0,0 +1,93 @@
+import { registerWebComponents } from '@domg-wc/common-utilities';
+import { VlTooltipComponent } from '../tooltip/vl-tooltip.component';
+import { VlProgressBarComponent } from './vl-progress-bar.component';
+import { html } from 'lit';
+
+registerWebComponents([VlProgressBarComponent, VlTooltipComponent]);
+
+const mountDefault = (steps: string[]) => cy.mount(html`<vl-progress-bar .steps=${steps}></vl-progress-bar> `);
+
+describe('component vl-progress-bar - default', () => {
+    const steps = ['Step 1', 'Step 2', 'Step 3'];
+
+    beforeEach(() => {
+        mountDefault(steps);
+    });
+
+    it('should mount', () => {
+        cy.get('vl-progress-bar');
+    });
+
+    it('should be accessible', () => {
+        cy.injectAxe();
+        cy.checkA11y('vl-progress-bar');
+    });
+
+    it('should render steps correctly', () => {
+        cy.get('vl-progress-bar').shadow().find('.vl-progress-bar__step').should('have.length', steps.length);
+
+        steps.forEach((step, index) => {
+            cy.get('vl-progress-bar')
+                .shadow()
+                .find(`.vl-progress-bar__step:nth-child(${index + 1}) `)
+                .find('vl-tooltip')
+                .contains(step);
+        });
+    });
+
+    it('should emit vl-click-step event when a step is clicked', () => {
```

Comment:
Voor het emitten van een event te testen maken we gebruik van onze cypress util `createStubForEvent`, je vindt hier een voorbeeld van in oa de testen van het accordion component.

Reply:
gedaan.